src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMetadata.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { ACTIVITY_TARGETS } from '../../../../common/interactionTargets';
+import { PLACEHOLDER_USER } from '../../../../../constants';
+import ActivityStatus from '../../common/activity-status';
+import ActivityTimestamp from '../../common/activity-timestamp';
+import Avatar from '../../Avatar';
+import IconAnnotation from '../../../../../icons/two-toned/IconAnnotation';
+import UserLink from '../../common/user-link';
+
+import type { FeedItemStatus } from '../../../../../common/types/feed';
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../../common/flowTypes';
+import type { User } from '../../../../../common/types/core';
+
+import messages from '../messages';
+
+import './BaseCommentMetadata.scss';
+
+export interface BaseCommentMetadataProps {
+    annotationActivityLink?: React.Element<any> | typeof undefined;
+    created_at: string | number;
+    created_by: User;
+    getAvatarUrl: GetAvatarUrlCallback;
+    getUserProfileUrl?: GetProfileUrlCallback | typeof undefined;
+    status?: FeedItemStatus | typeof undefined;
+}
+
+export const BaseCommentMetadata = ({
```

Comment:
We may want to reconsider the use of "metadata" and "data" in these components, since they're both generic and super overloaded. Maybe something like `BaseCommentCitation` or `BaseCommentByline`.

Reply:
Surprisingly tough to think of a good name here. This is less than a citation but more than a byline. Eventually, I asked ChatGPT, but tragically, I think its best suggestion was metadata. Another decent candidate, but still not great, is info... It's generic enough to capture author, timestamp (and sometimes activity link); but it's also kind of too generic.

Reply:
Would `BaseCommentSupplementalData` be any better?

Reply:
That's not a bad idea. I'll try that refactor right now, and see if I can keep everything working :)

Reply:
Actually, I think if we're going that generic, then `Info` is the same but more concise, and has the advantage of avoiding, `SupplementalData-data`. All pretty so-so names, but sometimes it's just hard to think of good ones.

Reply:
What about just calling this BaseCommentHeader instead of BaseCommentMetadata. This component is a UI construct that's more just a collection of ui elements(not necessarily comment data) that goes at the top of a comment.